### PR TITLE
18 Create conversation scripts to get user input

### PR DESCRIPTION
**Pull request by baha-a**

### CODE REVIEW

File: GUI/src/pages/SerivceSettingPage.tsx
Comment (trevorling) [inline GUI/src/pages/SerivceSettingPage.tsx:1]: Type in file name

File: GUI/src/pages/SerivceSettingPage.tsx [hunk 1]
```diff
@@ -0,0 +1,71 @@
+import React, { useEffect, useState } from "react";
+import { useTranslation } from "react-i18next";
+import { Button, FormInput, Track } from "../components";
+import { getServiceSettings, saveServiceSettings } from "../resources/api-constants";
+import axios from "axios";
+
+const SerivceSettingPage: React.FC = () => {
```
Comment (trevorling) [inline GUI/src/pages/SerivceSettingPage.tsx:7]: Please fix the typo.

File: docker-compose.yml [hunk 2]
```diff
@@ -4,7 +4,7 @@ services:
     container_name: ruuter
     image: ruuter
     environment:
-      - application.cors.allowedOrigins=http://localhost:3006, http://localhost:3001
+      - application.cors.allowedOrigins=http://localhost:3006
```
Comment (trevorling) [inline docker-compose.yml:7]: Why did you change it?

**Review by trevorling — CHANGES_REQUESTED** [review]

File: GUI/src/types/step.ts [hunk 3]
```diff
@@ -6,3 +6,18 @@ export interface Step {
   type: StepType;
   action?: string;
 }
+
+export enum StepType {
```
Comment (trevorling) [inline GUI/src/types/step.ts:10]: Redundant- duplicate.
It's in a separate file - `step-type.enum.ts`

Please remove it from here.

**Review by 1AhmedYasser — APPROVED** [review]

**Comment by 1AhmedYasser** [comment]
Please Resolve Conflicts

File: docker-compose.yml (hunk 2 above)
Comment (baha-a) [reply docker-compose.yml]: I did not changed it. I just need to merge main with this branch.

**Review by trevorling — APPROVED** [review]

File: DSL/Ruuter/GET/domain-file.yml [hunk 4]
```diff
@@ -21,4 +21,5 @@ convertYamlToJson:
   result: domainData
 
 return_value:
+  wrapper: false
```
Comment (Minwasko) [inline DSL/Ruuter/GET/domain-file.yml:24]: Please revert this change, or make sure that other endpoints using it will work with `wrapper: false`

**Review by Minwasko — CHANGES_REQUESTED** [review]

File: DSL/Ruuter/GET/domain-file.yml (hunk 4 above)
Comment (baha-a) [reply DSL/Ruuter/GET/domain-file.yml]: Reverted.

**Review by Minwasko — APPROVED** [review]

**Pull request merged**